### DISCUSSION TITLE
how to implement on a touch screen when dragging?

### DISCUSSION THREAD

**Discussion by ljmerza** [jaames/iro.js#253] [Q&A]
i have "input:change" event set on the color wheel but when i drag the marker around the color wheel, this caseus the events to trigger over and over while dragging. i can directly select a color but not drag to a color. is this possible?

**Comment by Rumidom** [comment]
I have the same problem, I do a post to a API once the color is chosen, would be nice to know when the user releases the mouse button, or touch
